.github/workflows/pr-checks.yml

```
@@ -158,3 +158,34 @@ jobs:
           # Test that the app can show SDK version
           ./build/taskscpp --ditto-sdk-version
           echo "✅ SDK version command works"
+
+  kotlin-multiplatform:
+    name: Kotlin Multiplatform (ubuntu-24.04)
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v3
```

The `gradle-build-action` is deprecated, superseded by `gradle/actions/setup-gradle`

```suggestion
        uses: gradle/actions/setup-gradle@v4
```